modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -0,0 +1,86 @@
+import sdpTransform from 'sdp-transform';
+
+/**
+ * RTCSessionDescription implementation.
+ */
+export default class RTCSessionDescription {
+    /**
+     * RTCSessionDescription constructor.
+     * @param {object} [descriptionInitDict]
+     * @param {string} [descriptionInitDict.type] "offer" / "answer".
+     * @param {string|object} [descriptionInitDict.sdp] SDP string or
```

Comment:
What I mean is that we don't want to allow calling new RTCSessionDescription({type: 'offer', sdp: some_sdp_transform_object }); from lib-jitsi-meet, because no other WebRTC implementation allows that.

About renaming I mean to use different field for 'sdp-transform' object, so that it would not be confused with the original stringified 'sdp' field. Just like you've called it 'parsed' inside of the constructor. @saghul WDYT ?